src/Data/LLVM/BitCode/IR/Function.hs

```diff
@@ -334,6 +335,11 @@ finalizeStmt lkp = relabel (resolveBlockLabel lkp)
 
 -- Function Block Parsing ------------------------------------------------------
 
+-- | A bit saying whether a call or callbr instruction has an explicit type, see
+-- LLVM's CallMarkersFlags.
```

Comment:
Including a URL to `CallMarkersFlags`' definition would be helpful for future reference.